packages/map-template/src/components/Search/Search.scss

```diff
@@ -28,11 +25,61 @@
         display: grid;
         gap: var(--spacing-x-small);
         overflow: auto;
-        grid-auto-rows: min-content;
+        flex: 1 1 auto;
         margin-top: var(--spacing-x-small);
+
+        > * {
+            height: fit-content; /* fixing issue on mobile Safari causing them to bee too large in case of just one search result */
+        }
     }
 
-    @media (max-width: $desktop-breakpoint) {
-        padding: var(--spacing-medium) var(--spacing-x-small) var(--spacing-medium) var(--spacing-x-small);
+    &__scroll-buttons {
+        position: fixed;
+        left: calc(50% + 284px + 20px); /* Half viewport width + half modal width + gap */
+        bottom: var(--spacing-x-large);
+
+        mi-scroll-buttons::part(container) {
+            background-color: transparent;
+            border: none;
+            padding: 0;
+            gap: var(--spacing-medium);
+        }
+
+        mi-scroll-buttons::part(button) {
+            margin: 0;
+            width: var(--spacing-xxxx-large);
+            height: var(--spacing-xxxx-large);
+            background: white;
+            border: none;
+            border-radius: var(--spacing-x-small);
+
+            &:disabled {
+                opacity: 1;
+            }
+
+            &::after {
+                width: 100%;
+                height: 100%;
+                display: flex;
+                justify-content: center;
+                align-items: center;
+            }
+        }
+
+        mi-scroll-buttons::part(button-up)::after {
+            content: url('../../assets/arrow-up.svg');
+        }
+
+        mi-scroll-buttons::part(button-down)::after {
+            content: url('../../assets/arrow-down.svg');
+        }
+
+        mi-scroll-buttons::part(button-up):disabled::after {
+            content: url('../../assets/arrow-up-disabled.svg');
+        }
+
+        mi-scroll-buttons::part(button-down):disabled::after {
+            content: url('../../assets/arrow-down-disabled.svg');
+        }
```

Comment:
What do we need this for? 🤔

Reply:
It's overriding the default looks of the scroll buttons component.

Reply:
Wouldn't it be a good idea to update the component instead? I will have to use the same buttons for the legend, and we'll end up with repetitive code

Reply:
I was reluctant to break the current layout of the component just for this one case, but since we will be using them in another place, I am ready to make the leap 😄 I'll update the component.

Reply:
👍 Did some refactor.